openforcefield/topology/molecule.py

```diff
@@ -1888,64 +1887,185 @@ def from_smiles(smiles,
         else:
             raise Exception(
                 'Invalid toolkit_registry passed to from_smiles. Expected ToolkitRegistry or ToolkitWrapper. Got  {}'
-                .format(type(toolkit_registry)))
+                    .format(type(toolkit_registry)))
 
-    def is_isomorphic(
-            self, other,
-            compare_atom_stereochemistry=True,
-            compare_bond_stereochemistry=True,
+    @staticmethod
+    def are_isomorphic(
+            mol1, mol2, return_atom_map=False,
+            aromatic_matching=True,
+            formal_charge_matching=True,
+            bond_order_matching=True,
+            atom_stereochemistry_matching=True,
+            bond_stereochemistry_matching=True,
     ):
         """
-        Determines whether the molecules are isomorphic by comparing their graphs.
+        Determines whether the two molecules are isomorphic by comparing their graphs.
 
         Parameters
         ----------
-        other : an openforcefield.topology.molecule.FrozenMolecule
+        mol1 : an openforcefield.topology.molecule.FrozenMolecule or TopologyMolecule or nx.Graph()
```

Comment:
If nx.Graphs are given, it would be helpful to know here what custom attributes must be present in the obj for the function to work correctly.

Reply:
Good point! ill add it to the docstring.

Reply:
Did you add this? Just double checking that I didn't miss it.

Reply:
Yes sorry I forgot to push it should show now.